lib/pact_broker/pacts/provider_state_service.rb

```diff
@@ -14,7 +14,7 @@ class ProviderStateService
       def self.list_provider_states(provider)
         query = scope_for(PactPublication).eager_for_domain_with_content.for_provider_and_consumer_version_selector(provider, PactBroker::Pacts::Selector.latest_for_main_branch)
         query.all.flat_map do | pact_publication |
-          pact_publication.to_domain.content_object.provider_states
+          { "providerStates" => pact_publication.to_domain.content_object.provider_states, "consumer" => pact_publication.to_domain.consumer.name }
```

Comment:
Should we consider returning more information here.

Would it be useful for example, for the returned list in the api to contain the links to the latest pacts on the consumers main branch, which contain the provider states returned?

Reply:
A link might be a nice touch. Given it's just a template and doesn't require additional DB lookups it should be pretty speedy. But equally fairly straightforward to re-lookup using the HAL links so I think we could consider adding it later if people really need.